packages/cashier/src/Components/account-prompt-dialog.jsx

```diff
@@ -19,28 +17,23 @@ const AccountPromptDialog = ({
 
     const non_crypto_accounts = accounts_list.filter(x => !x.is_crypto);
     const non_crypto_currency = non_crypto_accounts.map(x => x.currency)[0];
-    const is_crypto = !!currency && isCryptocurrency(currency);
 
     return (
         <Dialog
             className='acc-prompt-dialog'
-            title={is_crypto ? localize('Switch account?') : localize('Switch to crypto account?')}
-            confirm_button_text={is_crypto ? localize('Switch account') : localize('Switch to crypto account')}
+            title={localize('Switch account?')}
+            confirm_button_text={localize('Switch account')}
             cancel_button_text={localize('Cancel')}
             onConfirm={onConfirm}
             onCancel={onCancel}
             is_visible={should_show}
         >
-            {is_crypto ? (
-                <Localize
-                    i18n_default_text='To deposit money, please switch to your {{currency_symbol}} account.'
-                    values={{
-                        currency_symbol: non_crypto_currency?.toUpperCase(),
-                    }}
-                />
-            ) : (
-                <Localize i18n_default_text='To deposit cryptocurrency, switch your account.' />
-            )}
```

Comment:
Removing these since showing a dialog to switch to crypto account from fiat when making a deposit is no longer needed

Reply:
@farrah-deriv Do we have any confirmation for that?

Reply:
@Bahar-FS as per this discussion https://deriv-group.slack.com/archives/C027BS87TKN/p1635223042002100